aws_iam_policies/aws_iam_role_external_permission.py

```diff
@@ -0,0 +1,163 @@
+import json
+from panther_oss_helpers import resource_lookup
+from panther_oss_helpers import BadLookup
+from botocore.exceptions import NoCredentialsError
+
+# This is a list of the account numbers included in the organization
+# Example:
+# accounts = [
+#   '123456789012',
+#   '123456789013'
+# ]
+
+# account *12 is used as an organizational account in built-in unit tests
+# account *13 is used as a non-organizational account
+accounts = [
+    '123456789012',
+]
+
+# The specific permission that the policy checks
+# CONFIGURATION_REQUIRED: replace default "lambda:AddPermission" in unit tests
+#  with specified permission
+PERMISSION = 'lambda:AddPermission'
+
+# CONFIGURATION_REQUIRED: modify policy to contain specified permission, above
+mock_policy_has_permission = json.loads('''
+{
+    "PolicyDocument": {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "lambda:AddPermission",
+                    "lambda:GetPolicy"
+                ],
+                "Resource": "*"
+            }
+        ]
+    }
+}
+''')
+
+mock_policy_no_permission = json.loads('''
+{
+    "PolicyDocument": {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": "lambda:ListAliases",
+                "Resource": "*"
+            }
+        ]
+    }
+}
+''')
+
+
+# check to see if the account that is granted permission is third party
+def check_account(resource):
+    content_assumerole = resource.get('AssumeRolePolicyDocument')
+    if isinstance(content_assumerole, str):
+        content_assumerole = json.loads(content_assumerole)
+    print('json loaded content_assumerole is:\n\n',
```

Comment:
perhaps add newline before print statement to help with readability
```suggestion

    print('json loaded content_assumerole is:\n\n',
```